hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/mapreduce/TestTableOutputFormat.java

```diff
@@ -127,4 +128,21 @@ public void testTableOutputFormatWhenWalIsOFFForDelete()
     Assert.assertEquals("Durability of the mutation should be SKIP_WAL", Durability.SKIP_WAL,
       delete.getDurability());
   }
+
+  @Test
+  public void testOutputCommitterConfiguration() throws IOException, InterruptedException {
+    // 1. Verify it returns the default committer when the property is not set.
+    conf.unset(TableOutputFormat.OUTPUT_COMMITTER_CLASS);
+    tableOutputFormat.setConf(conf);
+    Assert.assertEquals("Should use default committer",
+      TableOutputCommitter.class,
+      tableOutputFormat.getOutputCommitter(context).getClass());
+
+    // 2. Verify it returns the custom committer when the property is set.
+    conf.set(TableOutputFormat.OUTPUT_COMMITTER_CLASS, CUSTOM_COMMITTER.getClass().getName());
+    tableOutputFormat.setConf(conf);
+    Assert.assertEquals("Should use custom committer",
+      CUSTOM_COMMITTER.getClass(),
+      tableOutputFormat.getOutputCommitter(context).getClass());
```

Comment:
Can you create custom committer for the test since the class won't change for the CUSTOM_COMMITTER ?